nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-5-processors/src/test/java/org/apache/nifi/processors/elasticsearch/TestPutElasticsearch5.java

```diff
@@ -36,6 +36,7 @@
 import org.elasticsearch.common.io.stream.StreamInput;
 import org.elasticsearch.common.settings.Settings;
 import org.elasticsearch.common.transport.TransportAddress;
+import org.elasticsearch.index.engine.VersionConflictEngineException;
```

Comment:
I see this imported but not used, Travis hasn't finished building but I expect some CheckStyle violations here and in the other file(s). Please run your maven build of this bundle with `-Pcontrib-check` , this will let you know if there are formatting violations, files without the correct headers, etc. Thanks!

Reply:
Removing this, and running the maven build with that flag ! :)